web/src/hooks/useLimits.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
+
+import { getLimits } from 'services/circuitbreaker';
+
+const useLimits = () => {
+  const query = useQuery<Limits, AxiosError<APIError>>({
+    queryKey: ['limits'],
+    queryFn: getLimits,
+    staleTime: 0,
+    refetchInterval: 30000,
```

Comment:
I noticed that it also refreshes if you switch back to the browser window. That's pretty cool.